requirements.txt

```diff
@@ -1,2 +1,8 @@
-Django==1.7.3
+Django >= 1.4, < 1.5
```

Comment:
Tests don't run properly on travis in these older versions of Django, as you can see from the _dramatic_ test coverage drop! I think we need to split up the requirements.txt file into requirements.txt, and testrequirements.txt, adding the newer Django version to testrequirements.txt so that it can run the tests appropriately.